packages/ui-components/style/icons/statusbar/bell.svg

```diff
@@ -0,0 +1,5 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="16" viewBox="0 0 16 16" version="1.1">
+   <path class="jp-icon2 jp-icon-selectable" stroke="#000000"
+      style="fill:none;stroke-width:1;stroke-linecap:butt;stroke-linejoin:miter;stroke-miterlimit:4;stroke-dasharray:none;stroke-opacity:1"
+      d="m 6.4730368,13.184615 c 0,0 0.073741,1.213254 0.5089876,1.526964 0.5505502,0.396815 1.4854021,0.396815 2.035952,0 0.4352479,-0.313707 0.5089881,-1.526964 0.5089881,-1.526964 m -8.65279535,0 H 15.125831 c 0,0 -1.171498,-0.520125 -1.526962,-1.017976 C 11.661919,9.4538174 13.155703,5.0830903 11.053928,2.4958678 10.336302,1.6124902 9.1381325,0.96890399 8.0000006,0.96890399 c -1.1381317,0 -2.3363013,0.64358611 -3.0539277,1.52696381 -2.1017733,2.5872225 -0.6079916,6.9579496 -2.54494,9.6707712 -0.3554643,0.497851 -1.52696375,1.017976 -1.52696375,1.017976 z" />
```

Comment:
An alternative to adding new styles for stroke would be to use this version (I also modified the stroke width to match that of other icons which is 35 mm):

```suggestion
   <path class="jp-icon2 jp-icon-selectable" fill="#333333"
      d="m8 0.29c-1.4 0-2.7 0.73-3.6 1.8-1.2 1.5-1.4 3.4-1.5 5.2-0.18 2.2-0.44 4-2.3 5.3l0.28 1.3h5c0.026 0.66 0.32 1.1 0.71 1.5 0.84 0.61 2 0.61 2.8 0 0.52-0.4 0.6-1 0.71-1.5h5l0.28-1.3c-1.9-0.97-2.2-3.3-2.3-5.3-0.13-1.8-0.26-3.7-1.5-5.2-0.85-1-2.2-1.8-3.6-1.8zm0 1.4c0.88 0 1.9 0.55 2.5 1.3 0.88 1.1 1.1 2.7 1.2 4.4 0.13 1.7 0.23 3.6 1.3 5.2h-10c1.1-1.6 1.2-3.4 1.3-5.2 0.13-1.7 0.3-3.3 1.2-4.4 0.59-0.72 1.6-1.3 2.5-1.3zm-0.74 12h1.5c-0.0015 0.28 0.015 0.79-0.74 0.79-0.73 0.0016-0.72-0.53-0.74-0.79z" />
```

Reply:
Thanks